packages/desktop-client/src/components/Modals.tsx

```diff
@@ -281,17 +335,22 @@ export function Modals() {
 
         case 'rollover-budget-summary':
           return (
-            <RolloverBudgetSummary
+            <NamespaceContext.Provider
```

Comment:
We need to think about how to remove the need to wrap modal components with this in order for `useSheetValue` to work. By default, there is no namespace context for any of our modals.